php-8.4-mysql/Dockerfile

```diff
@@ -0,0 +1,20 @@
+ARG ARCH=amd64
+FROM sykescottages/php:8.4-cli-$ARCH
+
+ENV TZ=Europe/London
+RUN ln -snf /usr/share/zoneinfo/$TZ /etc/localtime && echo $TZ > /etc/timezone
+
+RUN apt-get -qq update && \
+    apt-get -qq install -y \
+    php8.4-mysql \
+    php8.4-mcrypt \
+    php8.4-gd \
+    php8.4-ldap \
+    php8.4-amqp \
+    mysql-client
+
+COPY --from=composer:2 /usr/bin/composer /usr/bin/composer
```

Comment:
Not needed, it's inheriting from the `sykescottages/php:8.4-cli-$ARCH` which in turn inherts from the PHP 8 Base https://github.com/SykesCottages/docker-php/blob/master/8-base/Dockerfile#L11